### PR TITLE
Add script for preparing the DB for convenience

### PR DESCRIPTION
**Pull request by da-kami**

### CODE REVIEW

File: daemon/prepare_db.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+DAEMON_DIR=daemon
+TEMPDB=tempdb
+
+# crated temporary DB
```
Comment (klochowicz) [inline daemon/prepare_db.sh]: nit: typo `create'

Comment (klochowicz) [inline daemon/prepare_db.sh]: nit: typo `create'

File: daemon/prepare_db.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (klochowicz) [inline daemon/prepare_db.sh:1]: nit: I think you forgot to `chmod +x` this script

**Review by klochowicz — APPROVED** [review]
nice, it's always easier to maintain and use a script 👍 

shall we point to the script in the README so devs could use that instead of following the manual process?

File: daemon/prepare_db.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# create temporary DB
+DATABASE_URL=sqlite:$TEMPDB cargo sqlx database create
+
+# run the migration scripts to create the tables
+DATABASE_URL=sqlite:$TEMPDB cargo sqlx migrate run
+
+# prepare the sqlx-data.json rust mappings
+DATABASE_URL=sqlite:./$DAEMON_DIR/$TEMPDB cargo sqlx prepare -- --bin taker
+
+rm $TEMPDB
```
Comment (thomaseizinger) [inline daemon/prepare_db.sh]: You may want to do is in a `trap` to make sure it always runs on exit.

**Review by thomaseizinger — APPROVED** [review]
Thanks!

Should we specify `set -e` to bail early if something fails?

**Pull request merged**